lib/ansible/modules/cloud/azure/azure_rm_virtualmachine.py

```diff
@@ -1219,7 +1206,7 @@ def exec_module(self, **kwargs):
                                        "from the marketplace. - {2}").format(self.name, self.plan, str(exc)))
 
                     self.log("Create virtual machine with parameters:")
-                    self.create_or_update_vm(vm_resource, self.remove_autocreated)
+                    self.create_or_update_vm(vm_resource, self.remove_on_absent == 'all_autocreated')
```

Comment:
again, if default is `all_autocreated`, should it be called out somewhere, such as doc?

Reply:
well, not default, at least not yet....